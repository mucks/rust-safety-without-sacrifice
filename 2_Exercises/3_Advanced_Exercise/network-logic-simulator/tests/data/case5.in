# Consensus + async timing
NODE V1 weight=5 role=Validator
NODE V2 weight=3 role=Validator
NODE V3 weight=2 role=Validator
VOTE V1 for=Alice round=7
VOTE V2 for=Alice round=7
VOTE V3 for=Bob round=7
CONSENSUS round=7 quorum=66
ASYNC_SYNC ingest delay=300
ASYNC_SYNC analyze delay=100
ASYNC_SYNC archive delay=200
RUN_ASYNC
