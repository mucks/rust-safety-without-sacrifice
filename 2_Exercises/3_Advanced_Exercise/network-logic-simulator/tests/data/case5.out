CONSENSUS r7: leader=Alice quorum_met=true votes={Alice:2, Bob:1}
ASYNC done: analyze@100ms
ASYNC done: archive@200ms
ASYNC done: ingest@300ms
